File: eTheater.Services/Database/TheaterInfo.cs
using System;
using System.Collections.Generic;

namespace eTheater.Services.Database;

public partial class TheaterInfo : BaseEntity
{
    public int TheaterInfoId { get; set; }

    public string? Name { get; set; }

    public string? Logo { get; set; }

    public string? Adress { get; set; }

    public string? Email { get; set; }

    public string? Webpage { get; set; }

    public string? PhoneNumber { get; set; }
}

File: UI/etheater_mobile/lib/models/theaterInfo.dart
import 'package:json_annotation/json_annotation.dart';

part 'theaterInfo.g.dart';

@JsonSerializable()
class TheaterInfo {
  int theaterInfoId;
  String name;
  String? logo;
  String? adress;
  String? email;
  String? webpage;
  String? phoneNumber;
  bool? isDeleted;

  TheaterInfo(
      {required this.theaterInfoId,
      required this.name,
      this.logo,
      this.adress,
      this.email,
      this.webpage,
      this.phoneNumber,
      this.isDeleted});

  factory TheaterInfo.fromJson(Map<String, dynamic> json) =>
      _$TheaterInfoFromJson(json);
  Map<String, dynamic> toJson() => _$TheaterInfoToJson(this);
}
